src/reporter.js

```diff
@@ -29,12 +32,105 @@ class Reporter {
             accessKey: this.accessKey,
             headers: {'User-Agent': `cypress-reporter/${reporterVersion}`}
         });
+
+        this.testRunsAPI = new TestRunsAPI({
```

Comment:
Any benefit to populating the user agent header here as well? 🤔

Reply:
Good question, I'm not sure! Seems like the kind of thing future us will be thankful for though, so I'll add it now. I don't think its being set in the other reporters, so I'll make a ticket to add it there too.